nixos/modules/virtualisation/nixos-containers.nix

```diff
@@ -545,6 +549,16 @@ in
               '';
             };
 
+            systemCallFilter = mkOption {
+              type = types.listOf types.str;
+              default = [];
+              example = [ "add_key" "keyctl" "bpf" ];
+              description = lib.mdDoc ''
+                Set the system call filter for the container.  See the
```

Comment:
nit:

```suggestion
                Set the system call filter for the container. See the
```

Reply:
I just copied the double space from the above:
```nix
additionalCapabilities = mkOption {
  type = types.listOf types.str;
  default = [];
  example = [ "CAP_NET_ADMIN" "CAP_MKNOD" ];
  description = lib.mdDoc ''
    Grant additional capabilities to the container.  See the
    capabilities(7) and systemd-nspawn(1) man pages for more
    information.
  '';
};
```
I hence just assumed Nix manual pages followed [classic English spacing](https://en.wikipedia.org/wiki/History_of_sentence_spacing).

Reply:
Let me fix it for both

Reply:
> classic English spacing

Whoa, today I learned

The nix manual pages doesn't even follow it's own literal spacing. Everything is commonmark and a newline isn't a newline until there's two of them.